libvmi/driver/kvm/kvm.c

```diff
@@ -1364,6 +1380,17 @@ kvm_init_vmi(
     }
     vmi->num_vcpus = info.nrVirtCpu;
 
+#ifndef HAVE_LIBVMI_REQUEST
+    char *version = exec_info_version(kvm);
+    dbprint(VMI_DEBUG_KVM, "--Checking QEMU version string : %s", version);
+    // find substring "vmi"
```

Comment:
I may have noticed an issue in where we are doing the check.

the KVM driver has 2 memory accesses:
- the slow one, using QEMU commands
- the fast one, using the socket and the modified QEMU

Here, we are always failing if we detect that QEMU is `v2.8.0` or later.
So we deny the possibility of using the slow memory access.

Is it what you want, and focus only on the modified QEMU ?
Did i misunderstood something ?